app/src/components/v-form/v-form.vue

```diff
@@ -54,6 +54,7 @@
 				:validation-error="validationErrors.find((err) => err.field === field.field)"
 				:badge="badge"
 				@update:model-value="setValue(field, $event)"
+				@update:other-value="setOtherFieldValue($event)"
```

Comment:
I think we could reuse `setValue` here, like 

```
setValue($event.field, $event.value)
```

Reply:
Fixed by last commit, i reused the entire setValue method and changed a bit how other methods were calling it. We're now passing a fieldName and not a Field object. 
To explain it a bit, i had to change it to a field name because $event.field is not the entire field object but only its name, otherwise I had to keep my second method that retrieves a field from its name and then call the setValue method.